### PR TITLE
Implement elevated_permissions method

### PR DESCRIPTION
**Pull request by Haziba**
**Notes to tester:**
None

### CODE REVIEW

**Comment by stephenbinns** [comment]
Looks fine to me do you want to add to the examples on the readme?

**Comment by Haziba** [comment]
@stephenbinns :mag:

File: README.md [hunk 1]
```diff
@@ -100,6 +100,63 @@ $delete = $cronofy->delete_event($params);
...
+			'calendar_id' => 'calendarID_1',
+			'permission_level' => 'unrestricted'
+		),
+	array(
+			'calendar_id' => 'calendarID_2',
+			'permission_level' => 'unrestricted'
+		)
+);
+
+$delete = $cronofy->elevated_permissions($params);
```
Comment (stephenbinns) [inline README.md]: Probably wants to be `$response` similar below

**Comment by stephenbinns** [comment]
👍

**Pull request merged**